src/components/article/ViewArticle.jsx

```diff
@@ -0,0 +1,127 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+
+import { getArticle } from '../../actions/articles';
+import NavBar from '../presentation/NavBar';
+import DisplayMessage from '../presentation/DisplayMessage';
+import './ViewArticle.scss';
+import Footer from '../presentation/Footer';
+
+class ViewArticle extends React.Component {
+  state={
+    loading: 'https://i.imgur.com/ungt2Pg.gif',
+  };
+
+  componentDidMount = async () => {
+    const { match: { params }, getArticle: viewArticle } = this.props;
+    await viewArticle(params.slug);
+    this.setState({ loading: '' });
+  }
+
+  render() {
+    const { loading } = this.state;
+    const {
+      tags,
+      message,
+      article,
+      author,
+    } = this.props;
+
+    const {
+      title,
+      body,
+      imgUrl,
+      readtime,
+      createdAt,
+    } = article;
+    const { username } = author;
+    const date = new Date(createdAt).toDateString();
+    return (
+      <div>
+        <NavBar />
+        <div className="img-container">
+          <div>
+            <img src={!imgUrl ? process.env.REACT_APP_DEFAULT_ARTICLE_IMAGE : imgUrl} alt={!imgUrl ? null : 'article'} />
```

Comment:
So based on the ternary op in the `alt` field, if there's no `imgUrl`, there wouldn't be any alt text?

Reply:
Nice observation, I'll look into that @akhilome 👍